apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -326,6 +326,7 @@ export class PublishAction extends BaseRushAction {
         const registryUrl: string = this._registryUrl.value;
         env['npm_config_registry'] = registryUrl; // tslint:disable-line:no-string-literal
         registry = registryUrl.substring(registryUrl.indexOf('//'));
+        args.push(`--registry ${registryUrl}`);
```

Comment:
>${registryUrl} [](start = 30, length = 14)

This is a separate arg, so normally it should be pushed separately.  Perhaps execCommand() joins everything with spaces and lets the shell reparse it, but that may not always be the case